ddtrace/tracer/sampler.go

```diff
@@ -160,7 +160,8 @@ func (ps *prioritySampler) readRatesJSON(rc io.ReadCloser) error {
 // getRate returns the sampling rate to be used for the given span. Callers must
 // guard the span.
 func (ps *prioritySampler) getRate(spn *Span) float64 {
-	key := "service:" + spn.service + ",env:" + spn.meta[ext.Environment]
+	env, _ := getMeta(spn, ext.Environment)
+	key := "service:" + spn.service + ",env:" + env
```

Comment:
I have conflicted feelings about these particular changes. The other one around the first finished looks perfect to me.